packages/itinerary-body/src/stories/components.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from "react";
```

Comment:
Can we rename this file to be more closely aligned with what it contains?

Reply:
Done!